client/src/banners/active-banner.tsx

```diff
@@ -150,9 +150,26 @@ function Banner(props: BannerProps) {
   );
 }
 
+export const MDN_BROWSER_COMPAT_REPORT_ID = "mdn_browser_compat_report";
 const DEVELOPER_NEEDS_ID = "developer_needs";
 const SUBSCRIPTION_ID = "subscription_banner";
 
+function MDNBrowserCompatReportBanner() {
+  return (
+    <Banner
+      id={MDN_BROWSER_COMPAT_REPORT_ID}
+      classname="developer-needs"
```

Comment:
Don't worry about it 😄 I must have added it to test something and forgot to remove it again 😞 There is no need to add this banner to Yari. It will have been discontinued by the time Yari goes live.